server/sonar-web/src/main/js/apps/issues/components/BulkChangeModal.js

```diff
@@ -240,15 +248,20 @@ export default class BulkChangeModal extends React.PureComponent {
       </div>
     </div>;
 
-  renderCheckbox = (field: string) =>
+  renderCheckbox = (field /*: string */) =>
     <Checkbox checked={this.state[field] != null} onCheck={this.handleFieldCheck(field)} />;
 
-  renderAffected = (affected: number) =>
+  renderAffected = (affected /*: number */) =>
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove parentheses around the parameter of this arrow function. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=javascript%3AS3524)